web/src/app/schedules/temp-sched/TempSchedAddShiftForm.tsx

```diff
@@ -18,6 +19,11 @@ const useStyles = makeStyles({
 export default function TempSchedAddShiftForm(): JSX.Element {
   const classes = useStyles()
   const [manualEntry, setManualEntry] = useState(false)
+  const [zone] = useURLParam('tz', 'local')
+  const now = DateTime.local()
+    .setZone(zone)
+    .startOf('day')
+    .toFormat("yyyy-MM-dd'T'HH:mm:ss")
```

Comment:
use ISO format, this is missing the offset so it's going to have issues being interpreted as a minimum value.
```suggestion
    .toISO()
```

Reply:
I was finding that the min/max does not get recognized by native inputs unless in this specific format after having tried ISOs originally